time.md

```diff
@@ -667,7 +667,6 @@ func (p *Game) Finish(winner string) {
 
 // cli.go
 type CLI struct {
-	playerStore PlayerStore
 	in          *bufio.Reader
```

Comment:
The store is in the Game at this point, the constructor for CLI doesn't pass in a store.